New Syndicate Operative Base & ship

spockye opened the pull request:
<!-- Write **BELOW** The Headers and **ABOVE** The comments else it may not be viewable. -->
<!-- You can view Contributing.MD for a detailed description of the pull request process. -->

## About The Pull Request
This PR does two major things
 it replaces the current Nuclear operative base with a new and (my biased opinion) improved one, while also replacing the current (old_infiltrator) shuttle with the pre-existing (advanced_infiltrator)
The previous one was old. like. really old! while it has a lot of soul and im sure people are attatched to it, i think it might be time to replace it with something new. 

The new one is split into two parts. The base itself where you find your bar, gear room, mech chargers, drop pods, bathroom and arrivals.
on the second part you find the shuttle itself, located within its own hangar. you can get to and back from the hangar by using the elevator located south in the base. 

------


<!-- Describe The Pull Request. Please be sure every change is documented or this can delay review and even discourage maintainers from merging your PR! -->

## Why It's Good For The Game
the current base is very old, and lacks a lot of quality of life changes. while it does have the soul factor, it lacks everything else
<!-- Please add a short description of why you think these changes would benefit the game. If you can't justify it in words, it might not be worth adding.
-->

## Testing Photographs and Procedure
<!-- Include any screenshots/videos/debugging steps of the modified code functioning successfully, ideally including edge cases. -->
<details>
<summary>Screenshots&Videos</summary>


NEWEST UPDATE

![image](https://github.com/user-attachments/assets/47d9b06b-b746-4e89-8c7c-345d4051595f)


-------------

![image](https://github.com/BeeStation/BeeStation-Hornet/assets/79304582/34c8cee1-c561-42d4-b779-4a4aaff36006)

![image](https://github.com/BeeStation/BeeStation-Hornet/assets/79304582/6f31a4f8-e2fd-4987-b121-2a96546a2e5c)

![image](https://github.com/BeeStation/BeeStation-Hornet/assets/79304582/49bd093b-4590-48c3-a399-2859dd4c892e)

![image](https://github.com/BeeStation/BeeStation-Hornet/assets/79304582/2b046889-3520-4e04-8eab-896cf1303ac4)

![image](https://github.com/BeeStation/BeeStation-Hornet/assets/79304582/bb9d45f7-4b4e-4107-a1e4-6a908351152e)

![image](https://github.com/BeeStation/BeeStation-Hornet/assets/79304582/44bc4b6a-2d43-44ec-bf68-28f8324c2d38)

![image](https://github.com/BeeStation/BeeStation-Hornet/assets/79304582/ca81858f-94ed-4be0-afeb-8b92f31618b1)

![image](https://github.com/BeeStation/BeeStation-Hornet/assets/79304582/37ceff1e-0aa5-4a40-bb29-9954bb032a83)

![image](https://github.com/BeeStation/BeeStation-Hornet/assets/79304582/5f0bda7d-5502-451f-8a7c-82eeffb8e5e1)

--------------------------
Hangar with shuttle

![image](https://github.com/BeeStation/BeeStation-Hornet/assets/79304582/fef5f7de-e355-4dca-b1e8-3db7708959d7)

Hangar without shuttle

![image](https://github.com/BeeStation/BeeStation-Hornet/assets/79304582/6182a299-19fc-4843-afcd-ec73fca1d0ed)

Entire base

![image](https://github.com/BeeStation/BeeStation-Hornet/assets/79304582/90b29843-b07d-4bdf-85a7-3afefaa79a49)

Old base for refrence

![image](https://github.com/BeeStation/BeeStation-Hornet/assets/79304582/3e89e20f-a254-4373-b691-70c369641971)


</details>

## Changelog
:cl: @spockye SpaceSomalian
add: Replaces the current Nuclear Operative base 
add: replaces the syndicate infiltrator with the advanced infiltrator
add: ports improved syndicate posters and a syndicate S poster from shiptest
/:cl:

<!-- Both :cl:'s are required for the changelog to work! You can put your name to the right of the first :cl: if you want to overwrite your GitHub username as author ingame. -->
<!-- You can use multiple of the same prefix (they're only used for the icon ingame) and delete the unneeded ones. Despite some of the tags, changelogs should generally represent how a player might be affected by the changes rather than a summary of the PR's contents. -->

spockye committed:
it begins

Nukie base remake / new posters / poster updates / new syndicate pods

Tsar-Salat committed:
Drop Pod frontend

spockye committed:
trying out different hangars

spockye committed:
Update CentCom.dmm

spockye committed:
Update CentCom.dmm

spockye committed:
finishes hangar

also adds a bathroom + misc fixes

spockye committed:
finishes up both parts

spockye committed:
hangar without ship

spockye committed:
Merge remote-tracking branch 'upstream/master' into syndicate-asteroid

spockye committed:
Merge remote-tracking branch 'upstream/master' into syndicate-asteroid

spockye committed:
Update CentCom.dmm

spockye committed:
adds a missing light

spockye committed:
replaces all water with the AIR having version of water

PestoVerde322 commented on code/game/objects/effects/contraband.dm:
```diff
@@ -215,7 +215,7 @@
 	name = "Syndicate Recruitment"
 	desc = "See the galaxy! Shatter corrupt megacorporations! Join today!"
-	icon_state = "poster5"
+	icon_state = "poster_syndicate_recruitment"
```
Keep icon_state names as numbers for now since that's the standard, however for the future it might need a doublecheck on that.

spockye replied:
fixed!

Tsar-Salat commented on code/__DEFINES/cargo.dm:
```diff
@@ -49,19 +50,166 @@
+		POD_SHAPE_NORML,
+		"syndicate_droppod",
+		TRUE,
+		//"syndicate", //TODO: Door
```
Remember to replace this string with the name of the syndicate dropped door, after you add it to the supply pod.dmj

spockye committed:
Merge remote-tracking branch 'upstream/master' into syndicate-asteroid

spockye committed:
Merge remote-tracking branch 'upstream/master' into syndicate-asteroid

spockye committed:
responds to reviews

-fixes poster names
-fixes door names
-pixelshifted chairs

spockye committed:
updates more poster names

spockye committed:
Merge remote-tracking branch 'upstream/master' into syndicate-asteroid

spockye committed:
misc fixes

-fixes the advanced shuttle mobile and stat dock

-misc map fixes

-prepares centcom for the NEXT STAGE (at most, dubious)

spockye committed:
moves the base to the right location

more attempts at a temp elevator replacement

Tsar-Salat committed:
test open/close

spockye committed:
Merge remote-tracking branch 'upstream/master' into syndicate-asteroid

spockye committed:
is this a good idea? i dont know.    am i going to do it anyways? yes

lazyloading the syndie base. I may have to revert this later

spockye committed:
Revert "is this a good idea? i dont know.    am i going to do it anyways? yes"

This reverts commit 0e5a9b49299f00444e9001b7a06a44f7186d4b80.

spockye committed:
Merge remote-tracking branch 'upstream/master' into syndicate-asteroid

spockye committed:
death to elevators!!!

the elevator wont work, so?
im pretending like it doesnt exist!

(the elevator will be left where it is for later, but for now theres a new hallway leading to the shuttle!

spockye committed:
Update CentCom.dmm

spockye committed:
firelocks. decalchanges

spockye committed:
Merge remote-tracking branch 'upstream/master' into syndicate-asteroid

spockye committed:
small decal changes / four spawners instead of six

spockye committed:
adds escape shuttle dock

spockye committed:
adds the new catwalk tiles to the advanced infiltrator

spockye committed:
Merge remote-tracking branch 'upstream/master' into syndicate-asteroid

spockye committed:
stray decal

spockye committed:
Merge remote-tracking branch 'upstream/master' into syndicate-asteroid

spockye committed:
Merge remote-tracking branch 'upstream/master' into syndicate-asteroid

spockye committed:
Merge remote-tracking branch 'upstream/master' into syndicate-asteroid

spockye committed:
fixes some lattices in walls

spockye committed:
removes unused elevator junk

Tsar-Salat committed:
Merge remote-tracking branch 'BeeStation/master' into syndicate-asteroid

Tsar-Salat committed:
techfloor replacement

spockye committed:
Merge remote-tracking branch 'upstream/master' into syndicate-asteroid

spockye committed:
seeing red

(trash plate removal)

spockye committed:
misc fixes

spockye committed:
more small fixes

linter hell

Tsar-Salat committed:
Merge remote-tracking branch 'BeeStation/master' into syndicate-asteroid

Tsar-Salat committed:
manual edits to remove bad var

Tsar-Salat committed:
oops

Tsar-Salat committed:
droppod map & snowflake removal

spockye committed:
Merge remote-tracking branch 'upstream/master' into syndicate-asteroid

spockye committed:
Merge remote-tracking branch 'upstream/master' into syndicate-asteroid

spockye committed:
gives in to peer preassure

spockye committed:
Merge remote-tracking branch 'upstream/master' into syndicate-asteroid

spockye committed:
misc updates / pixel errors

spockye committed:
wth. snow??

spockye committed:
snows it up part two

Tsar-Salat committed:
Merge remote-tracking branch 'BeeStation/master' into syndicate-asteroid

Tsar-Salat committed:
fix

spockye committed:
Merge remote-tracking branch 'upstream/master' into syndicate-asteroid

spockye committed:
updates

spockye committed:
Update CentCom.dmm

spockye committed:
Update CentCom.dmm

spockye committed:
Update CentCom.dmm

spockye committed:
Merge remote-tracking branch 'upstream/master' into syndicate-asteroid

spockye committed:
Merge remote-tracking branch 'upstream/master' into syndicate-asteroid

spockye committed:
fixes conflict

spockye committed:
Update CentCom.dmm

spockye committed:
Merge remote-tracking branch 'upstream/master' into syndicate-asteroid

spockye committed:
lets see if this fixes it... edition 1

Tsar-Salat committed:
Merge remote-tracking branch 'BeeStation/master' into syndicate-asteroid

Tsar-Salat committed:
try to get centcom fixed

Tsar-Salat committed:
more roundstart active turfs

Tsar-Salat committed:
sorry marty

Tsar-Salat committed:
mapping helper

Tsar-Salat committed:
these turfs suck so bad

spockye committed:
Merge remote-tracking branch 'upstream/master' into syndicate-asteroid

spockye committed:
Merge remote-tracking branch 'upstream/master' into syndicate-asteroid

spockye committed:
removes pod sprite

no more pods!
we hate pods!
no more pods!
we hate pods!

spockye committed:
Revert "removes pod sprite"

This reverts commit f0de425f6d8dbf28f485d0f0d668df030d9ba342.
